src/api/index.tsx

```diff
@@ -191,6 +194,7 @@ export const getVersionsList = async ({
 export const logOutFromServer = async (apiState: ApiState) => {
   return callApi<{}>({
     apiState,
+    includeCredentials: true,
```

Comment:
Maybe it's just me but a comment would help explain why this is needed. It wasn't obvious to me before I researched this that you would need to *send* cookies to the server in order for the response to delete those cookies. I still don't know if this is the real problem, heh. We'll see.